### ISSUE TITLE
Plugins: Can't select business plugin only

### ISSUE THREAD

**Issue by v18** [Automattic/wp-calypso#825]
When selecting one of the Gumroad plugins,  it's not possible to select the business plugin or just the .org plugin in the manage mode, but instead both plugins get selected. 

See this gif:

![plugin-select-both-gumroad](https://cloud.githubusercontent.com/assets/11487924/11412835/9e74e92e-93af-11e5-8492-01fa7dd73819.gif)

```
<input type="checkbox" data-reactid=".3.2.1.1.$gumroad.0.0" readonly="" id="gumroad" class="plugin-item__checkbox">
<input type="checkbox" data-reactid=".3.2.2.1.$gumroad.0.0" readonly="" id="gumroad" class="plugin-item__checkbox">
````

The solution for now will be to disable the bulk edit mode on for business sites.

The two different checkboxes - one for the wpcom Gumroad plugin and the other for the w.org Gumroad plugin - have the same IDs, which is also problematic:

**Comment by rickybanister** [comment]
It looks like the solution here is to separate bulk edit. Each 'Edit All' button should only allow editing within its own scope (business or .org plugins). 

By treating each separately we will be able to avoid selections across the lists.

**Comment by enejb** [comment]
Fixed with #2296
